editor.planx.uk/src/@planx/components/Result/model.ts

```diff
@@ -34,3 +37,27 @@ export interface PresentationalProps {
   resetButton?: boolean;
   resetPreview?: () => void;
 }
+
+const overridesSchema = lazy(obj => 
+  object(
+    mapValues(obj, () => object({
+      heading: string(),
+      description: richText(),
+      resetButton: boolean().optional(),
+    }))
+  )
+);
+
+export const validationSchema = baseNodeDataValidationSchema.concat(object({
+  flagSet: mixed().oneOf([
+    "Planning permission", 
+    "Works to listed buildings", 
+    "Works to trees & hedges", 
+    "Demolition in a conservation area", 
+    "Planning policy", 
+    "Community infrastructure levy", 
+    "Material change of use",
+  ]),
```

Comment:
nit: is there a way to read this list from planx-core so we don't have to remember to update two places if adding a new flagset now?

Reply:
Good shout - fixed in d5b45607f038eab47d3af1a11712b798552963b6